### DISCUSSION TITLE
install big sur and catalina with Hacmini3.15 failure with stack .

### DISCUSSION THREAD

**Discussion by kingrhy** [osy86/HaC-Mini#1009] [Q&A] [answered]
i was install and running catlinia over a year. install smoothly and work fine(no audio) with Hacmini 3.01. 
recently I just want try to fix audio problem. and wants upgrade to 12.3. but when I try to. re-install .i encounter may problem . i cannot install OSx in NUc8i7hvk anymore so far.

this is error i encuntered:

during try to install catlina with 3.15, I got below message and lost HDMI signal ( computer seems still work but monitor no display and show message said not detected signal from computer,will into power save mode). I must push power button to power off . problem will repeatly.
<img width="974" alt="截屏2022-05-04 22 21 44" src="https://user-images.githubusercontent.com/16399048/166859083-a4426ffc-e297-495e-b566-8da900ae8a1b.png">

below is error during I install big sur with Hacmini 3.15. 
![bigsur with 315](https://user-images.githubusercontent.com/16399048/166859155-311c5108-113c-4d80-b3fd-dd5e20aa3a92.jpg)

if not plug network cable ,then i got below message while install bigsur with 3.15, just only tiny diffirent:

![111651724286_ pic](https://user-images.githubusercontent.com/16399048/166863199-8a76e866-134a-488c-b4f5-da9a24f484ad.jpg)

**Comment by kingrhy** [comment]
both install has diffirent message and but issue same: which that both cause monitor into black screen but computer not restart or shutdown.  just no signal input

**Comment by osy** [comment]
Is this a fresh install (formatted drive)? Please post your device configuration and the hardware you’ve installed (wifi, ssd, etc).

**Reply by kingrhy** [reply]
@osy  problem has been fix，after I post this issue,my nuc completely dead. then I found a professional guy and let him check and try to fix. and he told me cpu Pseudo Soldering .he re-welding that cpu and notice bios crash as well(this may because of I restart many times).after re-burn bios.nuc back to good. last week i see some one posted in a china x86 mac community said he was encouter similar issue but he fix that by switch another display port. So i guess my NUC dead because i never turn it off for at least 3-4month and I was running some heave video decoding work for at least 4 days.i guess cpu already overheat and
 Cuase cpu false soldering but it finally dead when i bring it home and connect to 4k
Monitor and try to re-install macOS
anyway its back to normal seems everything good i can even install Montery smoothly and almost perfect.


Only one thing still confuse me:now i use it in my office and connected to a 2k monitor.it work fine.i still don’t know if it can work on 4k monitor 

BTW: I want re-install os because I want fix audio issue.(audio not work completely front and rear ports).but when I received fixed NUC and re-install MacOS 12.4. (With 3.15),all audio ports work good .
we can close this thread 
thank you for care about this @osy

**Comment by kingrhy** [comment, marked as answer]
fix myNUC spend about $25 (CNY ¥150).  It is cheaper than buy a brand new. But i wait three week because of this guy do this job as his part-job. he only use weekend time to test and fix.

**Reply by xiaooloong** [reply]
老哥你哪里修的，求推荐。现在停产京东都只退不修了。

**Reply by kingrhy** [reply]
海鲜市场搜ID： laicelc   浙江温州的  B站上他的名字叫： 维客数码，不过一般不回复。还是闲鱼找他。价格个人觉得也比较靠谱。

**Comment by kingrhy** [comment]
update 2 month later.  i notice some screen Flickering if i use Resolution scaling。switch to other resolution and switch it back can fix short time. i guess its possible not fix completely .but its a tiny impact to use. so the workaround is switch resolution when screen filckering.
